respy/method_of_simulated_moments.py

```diff
@@ -99,8 +104,6 @@ def get_moment_errors_func(
     ValueError
         If the number of functions to compute the simulated moments does not match the
         number of empirical moments.
-    ValueError
-        If simulated moments and the comparison plot data should be returned.
 
     """
     empirical_moments = copy.deepcopy(empirical_moments)
```

Comment:
Maybe we could use different names for internal and external moments maybe. I think that makes things more explicit here.

Reply:
Yes that might be a good idea now that we have multiple outputs! I'll make a suggestion